config/m3aggregator.yaml

```diff
@@ -76,10 +77,6 @@ aggregator:
           capacity: 32
         - count: 1024
           capacity: 64
-  timerQuantiles:
```

Comment:
We can't, as we don't support setting timerquantiles in the options now, and quantiles needs to be valid aggregation types anyway so we should just set aggregation types